package.json

```diff
@@ -76,35 +76,6 @@
       "<rootDir>/node_modules/(?!(@react-native|react-native|react-navigation))"
     ]
   },
-  "resolutions": {
-    "@ledgerhq/hw-transport-node-hid": "file:./patches/fake-hw-transport-node-hid",
-    "axios": "0.21.1",
-    "babel-runtime": "6.26.0",
-    "bip39": "3.0.2",
-    "bn.js": "4.11.8",
-    "bs58": "4.0.1",
-    "bs58check": "2.1.2",
-    "clipboardy": "3.0.0",
-    "core-js": "2.5.3",
-    "create-hash": "1.2.0",
-    "elliptic": "6.5.4",
-    "eosjs-api": "https://github.com/EdgeApp/eosjs-api.git#prepare-script",
-    "hash-base": "3.0.4",
-    "hdkey": "1.1.1",
-    "inherits": "2.0.4",
-    "long": "4.0.0",
-    "msw": "1.3.2",
-    "process-nextick-args": "2.0.0",
-    "randombytes": "2.1.0",
-    "regenerator-runtime": "0.13.2",
-    "ripemd160": "2.0.2",
-    "ripple-binary-codec": "0.2.3",
-    "safe-buffer": "5.1.2",
-    "scrypt.js": "0.3.0",
-    "sha.js": "2.4.11",
-    "tweetnacl": "^1.0.3",
-    "uuid": "3.3.2"
-  },
```

Comment:
Why "Remove package.json resolutions"?

Reply:
Back when accountbased, exchange-plugins, and currency-plugins would be bundled together inside the GUI build process, we would use these resolutions to de-duplicate things. However, we were somewhat cheating by substituting semver-major changes. So if accountbased wants bip39 v3, and currency-plugins wants bip39 v4 we would just force both to use buffer v4, possibly breaking one or the other. On the other hand, with bip39 specifically being several 100K in size, this was a nice size win.

Now that accounbased and exchange-plugins provide their own pre-bundled build output, these resolutions don't do anything for our output size, but they do mean we are stuck with old & miss-matching package versions, which is no good.